apps/3000-home/.env

```diff
@@ -0,0 +1 @@
+NEXT_PRIVATE_LOCAL_WEBPACK=true
```

Comment:
Must tell next to use webpack from node modules, not its internal subset of webpack